lib/Middleware/SessionMiddleware.php

```diff
@@ -171,6 +173,8 @@ public function afterException($controller, $methodName, \Exception $exception):
 		}
 
 		if ($exception instanceof InvalidSessionException) {
+			// We spotted occurrences of this in combination with out-of-sync situations, so let's log it for now.
+			$this->logger->error('Invalid session: ' . $exception->getMessage(), ['exception' => $exception]);
```

Comment:
I think we can do this more specific and put it in here:
https://github.com/nextcloud/text/blob/main/lib/Middleware/SessionMiddleware.php#L94 as that's the function that's checked on push requests to a non public session and would throw an `InvalidSessionException`.

That would leave out exceptions caused elsewhere from the logs.

Reply:
Good point, pushed a change now. Is this how you meant it?

Reply:
exactly!